src/Cli/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -16,7 +16,8 @@ namespace Microsoft.DotNet.Cli.Utils
     internal class MSBuildForwardingAppWithoutLogging
     {
         private static readonly bool AlwaysExecuteMSBuildOutOfProc = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_RUN_MSBUILD_OUTOFPROC");
-        private static readonly bool UseMSBuildServer = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_USE_MSBUILD_SERVER", false);
+        private static readonly bool UseMSBuildServerEnvVarConfig = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_USE_MSBUILD_SERVER", true)
```

Comment:
Why does this have to be more complicated? That is, why would anyone have both DOTNET_CLI_USE_MSBUILD_SERVER set to true and DOTNET_CLI_DO_NOT_USE_MSBUILD_SERVER set to true?

Reply:
Since now DOTNET_CLI_USE_MSBUILD_SERVER is true by dafault, people who want to dissable it migth use DOTNET_CLI_DO_NOT_USE_MSBUILD_SERVER . 
Unfortunately we have officially documented that the way to not use MSBuild server is through DOTNET_CLI_DO_NOT_USE_MSBUILD_SERVER. We will remove it from docs and will recommend only DOTNET_CLI_USE_MSBUILD_SERVER but Chet mentioned that had respect this env var if people set it for any reason. Maybe we can delete it in year or so, but I am afraid it will stick.